packages/modules/viewer-react/src/components/app-ui/providers/DefaultContentGroupProvider.ts

```diff
@@ -0,0 +1,67 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { StandardContentLayouts } from "@itwin/appui-abstract";
+import {
+  ContentGroup,
+  ContentGroupProvider,
+  IModelViewportControl,
+  UiFramework,
+} from "@itwin/appui-react";
+
+import { getAndSetViewState } from "../../../services/iModel";
+import type {
+  BlankConnectionViewState,
+  ViewerViewCreator3dOptions,
+  ViewerViewportControlOptions,
+} from "../../../types";
+
+/**
+ * Provide a default content group to the default frontstage
+ */
+export class DefaultContentGroupProvider extends ContentGroupProvider {
+  private _viewportOptions: ViewerViewportControlOptions | undefined;
+  private _blankConnectionViewState: BlankConnectionViewState | undefined;
+  private _viewCreatorOptions: ViewerViewCreator3dOptions | undefined;
+
+  constructor(
```

Comment:
nit: i think we should start passing objects around instead of multiple args in the future, in next

Reply:
comment applies i think for a lot of the fns, classes where we have multiple args. but this is a huge nit, and something to tackle later

Reply:
I agree, but just need to be careful that there is no mutation (which there shouldn't be anyways but I have seen it in some code that we work with often :-)) and that the object isn't passed along too many times

Reply:
I hope not my code lol